### ISSUE TITLE
Questions about the experiment part

### ISSUE THREAD

**Issue by qfkkwgd** [mengqunhan/MWP-NAS#1]
1. In your Table 1, DeductReasoner (Jie et al, 2022) is used as a baseline. From their original paper, DeductReasonser with a RoBERTa encoder achieves 92% accuracy on MAWPS (5-fold cross validation). I wonder why your reproduced results only obtains 86%? It seems this is a huge performance drop. And I think you don't compare with your method with SOTA methods (though you mentioned you did so in section 5.1). Please see the baselines considered in (Jie et al, 2022), e.g. Roberta-GTS and Roberta-Graph2Tree from (Patel et al., 2021).
2. In your footnote 4 in page 7, you mentioned "In the main part of Jie et al 2022, the authors implement a different training setting, combining training and validation sets for their training." I couldn't find such statements in their paper. Where did you find it?

**Comment by YiBin-CHN** [comment]
Hi there, sorry we missed your issue these days. Pls kindly find the answers:

1. Because DeductReasoner (Jie et al, 2022) used different data split with ours. Our method is based on SUMC (Wang et al, 2022), so we implemented the experiments based on their provided data splits, and reproduced the results of DeductReasoner using their official codes for fair comparison. We didn't reproduced Roberta-GTS and Roberta-Graph2Tree because DeductReasoner significantly outperformed them as reported in Jie et al (2022), and our method outperformed DeductReasoner.

2. We contacted the authors and were told that.

Hope they are helpful.
